lib/status/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+var status = angular.module('guStatus', []);
+
+status.directive('guStatus', function statusDirective(){
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: (el, attrs) => {
```

Comment:
Yeah, Im kinda not sure about best practice here.  My assumption is that its more performant to format templates within this function rather than manipulating scope within the link function or controller, but does this adhere to the purpose of this function? Whats easier to grok?